server/app/routes/user/index.js

```diff
@@ -50,6 +32,52 @@ router.post('/', function(req, res, next) {
     }
 })
 
+// Get a user's info, including orders, addresses, billing for personal info page
+router.use('/info', function (req, res, next) {
```

Comment:
needed to move this before :orderId because express would always think that 'info' was an orderId and would give an error.